runtime/doc/eval.txt

```diff
@@ -5405,13 +5405,25 @@ server2client( {clientid}, {string})			*server2client()*
 			:echo server2client(expand("<client>"), "HELLO")
 <
 serverlist()					*serverlist()*
-		Return a list of available server names, one per line.
-		When there are no servers or the information is not available
-		an empty string is returned.  See also |clientserver|.
-		{only available when compiled with the |+clientserver| feature}
+		Return a list of available server names in a list.
```

Comment:
This might not be based on anything concrete, but I've always interpreted it this way.

Function definitions (or function docs in this case) shouldn't say what's going to happen immediately, but instead what will happen if they're called. Let's say that Jim eats food. When talking about Jim, you wouldn't say "Jim eat food" or "Jim is eating food", you'd say "Jim eats food". That is to say, Jim does indeed eat food, but is not always doing so (for the sake of the example).

Regarding functions, take the following example:

```c
// declare i and initialize it to 5
static int i = 5;

// return 5
int ret5(void) {
  return 5;
}
```

The first comment is correct, because it makes sense given that the program is read from top to bottom. The second one isn't, because unlike the variable above it, it's not executed when it's, it's just defined. `Returns 5` would be better as it just says what `ret5()` does if used.

I hope that didn't come across as condescending, as I'm just trying to be clear in my example.